projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/types.ts

```diff
@@ -0,0 +1 @@
+export type NoticeHookType = ( redBubbleAlerts: RedBubbleAlerts, isLoading: boolean ) => void;
```

Comment:
If you don't import anything in .ts file, I don't think you need to export. Just a small note.

Reply:
I've never heard of this, and it's very interesting 🤔 

However I'm getting a linting error when I try to remove the export

`'NoticeHookType' is defined but never used.eslint[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)`